lib/linguist/languages.yml

```diff
@@ -1606,6 +1606,34 @@ Dotenv:
   tm_scope: source.dotenv
   ace_mode: text
   language_id: 111148035
+Dune:
+  type: programming
+  ace_mode: lisp
+  filenames:
+  - dune
+  - dune-file
```

Comment:
This file is nowhere near popular enough for inclusion. Its existence in this PR will significantly delay it being merged whilst we wait for popularity to be met. I recommend removing it. Users can use an override until popularity is sufficient for addition later.

Reply:
I've now left out `dune-file`.